scripts/check-conformance.mjs

```
@@ -0,0 +1,489 @@
+ * Sub-checks (run one with `--only=<name>[,<name>]`):
+ *   listings-sync        new components must appear on the docs listing surfaces
+ *   docs-sections        API pages must have the house sections
+ *   scss-conformance     SCSS partials follow the register-vars pattern
+ *   story-per-component  every exported component module has a .stories.tsx
+ *   autodocs-tag         every story file opts into autodocs
+ *   inline-style         no NEW inline style={{}} in stories/docs (ratcheted
+ *                        against scripts/conformance-baseline.json; after
+ *                        removing styles, shrink it with `--update-baseline`)
+ */
+import { readFile, readdir, writeFile, access } from 'node:fs/promises';
+import { join, relative, dirname, basename } from 'node:path';
```

ESLint

[error] 'basename' is defined but never used.

(no-unused-vars)